src/core/api/public_api.ts

```diff
@@ -751,6 +765,17 @@ class Player extends EventEmitter<IPublicAPIEvent> {
     /** Emit playback events. */
     let playback$ : ConnectableObservable<IInitEvent>;
 
+    const speed$ = combineLatest([
+      this._priv_speeds.speed$,
+      this._priv_speeds.trickModeSpeed$,
+      this._priv_trickModeEnabled$,
```

Comment:
Isn't that risky because we would always need to check that `trickModeSpeed$` is set before `trickModeEnabled$`?

Why not a single behavior subject for trickModes, with a special value when disabled?